src/Forms/Android/ArcGISRuntime.Xamarin.Forms.Android.csproj

```diff
@@ -56,9 +56,11 @@
     <!-- References -->
     <Reference Include="Esri.ArcGISRuntime, Version=100.2.0.0, Culture=neutral, PublicKeyToken=8fc3cc631e44ad86, processorArchitecture=MSIL">
       <HintPath>..\..\..\packages\Esri.ArcGISRuntime.Xamarin.Android.100.2.0\lib\MonoAndroid10\Esri.ArcGISRuntime.dll</HintPath>
+      <Private>True</Private>
```

Comment:
I don't think any of these changes are relevant to the sample. We should remove these changes.